arelle/CntlrCmdLine.py

```diff
@@ -1017,6 +1017,10 @@ def monitorParentProcess():
                             startedAt = time.time()
                             if options.formulaAction: # don't automatically run formulas
                                 modelXbrl.hasFormulae = False
+                            self.addToLog(format_string(self.modelManager.locale,
```

Comment:
Was this meant to be committed? There are now multiple `validated in` messages, one of which is always 0.00 seconds:

```bash
> python arelleCmdLine.py --plugins transforms/SEC --validate --file https://www.sec.gov/Archives/edgar/data/1445305/000144530523000166/0001445305-23-000166-xbrl.zip
[info] Activation of plug-in SEC Inline Transforms successful, version 19.2. - transforms/SEC
[info] loaded in 0.61 secs at 2023-11-30T16:39:02 - https://www.sec.gov/Archives/edgar/data/1445305/000144530523000166/0001445305-23-000166-xbrl.zip/wk-20230930.htm
[info] validated in 0.00 secs - https://www.sec.gov/Archives/edgar/data/1445305/000144530523000166/0001445305-23-000166-xbrl.zip/wk-20230930.htm
[info] validated in 0.72 secs - https://www.sec.gov/Archives/edgar/data/1445305/000144530523000166/0001445305-23-000166-xbrl.zip/wk-20230930.htm
```